docs/network/edges.html

```diff
@@ -4,66 +4,115 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-    <meta charset="utf-8">
```

Comment:
Puh! You really should not change every line in this document! Please only add your lines. It's impossible to review like this.